Go/tests/syntax_test_go.go

```diff
@@ -4961,8 +4961,12 @@ by accident, but if necessary, such support could be sacrificed.
 
     break
 //  ^^^^^ keyword.control.flow.break.go
-    case
+    case no_colon_here_while_user_is_typing
 //  ^^^^ keyword.control.conditional.case.go
+        ; a := b
+//      ^ punctuation.terminator
+//          ^ - punctuation.separator.case-statement
+//          ^^ keyword.operator.assignment
```

Comment:
I'd leave those lines dedicated to keyword tests.

Line 5629ff. seems more appropriate to add higher level (incomplete) expression tests.

Reply:
moved the test :)